From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Wed, 4 Feb 2015 22:11:13 +0000
Subject: libgee 0.8

Build against libgee-0.8 instead of libgee-0.6 aka gee-1.0

See https://bugs.debian.org/753878 which reads:

src:pdf-presenter-console builds a binary which depends on libgee2
(libgee-dev), which is deprecated in favor of libgee-0.8-2
(libgee-0.8-dev). Please test your package against the new libgee and
update to it.
---
 c-src/CMakeLists.txt | 2 +-
 src/CMakeLists.txt   | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/c-src/CMakeLists.txt b/c-src/CMakeLists.txt
index 7b380f8..a1ae024 100644
--- a/c-src/CMakeLists.txt
+++ b/c-src/CMakeLists.txt
@@ -1,7 +1,7 @@
 find_package(PkgConfig)
 pkg_check_modules(GOBJECT REQUIRED gobject-2.0)
 pkg_check_modules(GIO REQUIRED gio-2.0)
-pkg_check_modules(GEE REQUIRED gee-1.0)
+pkg_check_modules(GEE REQUIRED gee-0.8)
 pkg_check_modules(POPPLER REQUIRED poppler-glib)
 pkg_check_modules(GTK REQUIRED gtk+-2.0)
 pkg_check_modules(GTHREAD REQUIRED gthread-2.0)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3495433..0a8f432 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,7 +1,7 @@
 find_package(PkgConfig)
 pkg_check_modules(GOBJECT REQUIRED gobject-2.0)
 pkg_check_modules(GIO REQUIRED gio-2.0)
-pkg_check_modules(GEE REQUIRED gee-1.0)
+pkg_check_modules(GEE REQUIRED gee-0.8)
 pkg_check_modules(POPPLER REQUIRED poppler-glib)
 pkg_check_modules(GTK REQUIRED gtk+-2.0)
 pkg_check_modules(GTHREAD REQUIRED gthread-2.0)
@@ -48,7 +48,7 @@ vala_precompile(VALA_C
 	${VALA_SRC}
 PACKAGES 
 	gio-2.0
-        gee-1.0
+        gee-0.8
 	poppler-glib
 	gtk+-2.0
 	posix
